daemon/daemon_main.go

```diff
@@ -602,6 +602,9 @@ func init() {
 	flags.MarkHidden(option.MaxCtrlIntervalName)
 	option.BindEnv(option.MaxCtrlIntervalName)
 
+	flags.Int(option.MetricsLevel, option.MetricLevelCore, "Level of metrics that Cilium exposes.")
```

Comment:
What about we  do a StringSlice and allow to define a mask of what should be recorded?

`--metrics=kvstore-histogram --metrics=bpf-syscall [...]`

Reply:
Agreed, this is more user-friendly, as the choices for integers are arbitrary.